scripts/ci/test_plan.py

```diff
@@ -375,8 +418,8 @@ def parse_args():
         print("\n".join(files))
         print("=========")
 
-
-    f = Filters(files, args.pull_request, args.platform)
+    f = Filters(files, args.ignore_path, args.alt_tags, args.testsuite_root,
+                args.pull_request, args.platform, args.detailed_test_id)
```

Comment:
Maybe would be better to just pass `args` to `Filter` class and handle with those parameters inside that class. It is not ideal solution, but here can be cleaner.
* you can move logic to get changed files from `__main__` to helper method in `Filter` class
* you do not need to use globals: `repository_path`, `repo_to_scan`, `args` (btw, instead of using `args`, you can only pass `args.commits`)